### PR TITLE
Map Image PDF Export

### PR DESCRIPTION
**Pull request by NielsCharlier**

### CODE REVIEW

File: core/src/main/resources/mapfish-spring-config-output-formats.xml [hunk 1]
```diff
@@ -24,6 +24,8 @@
         p:fileSuffix="jpeg" p:contentType="image/jpeg"/>
     <bean id="tiffMapOutputFormat" class="org.mapfish.print.output.MapExportOutputFormat" scope="prototype"
         p:fileSuffix="tiff" p:contentType="image/tiff"/>
-    <bean id="tifMapOutputFormat" class="org.mapfish.print.output.MapExportOutputFormat" scope="prototype"
+    <bean id="tifMapOutputFormat" class="org.mapfish.print.output.MapExportOutputFormat" scope="prototype"
```
Comment (sbrunner) [inline core/src/main/resources/mapfish-spring-config-output-formats.xml]: Extra endline white space :-(

Comment (NielsCharlier) [inline core/src/main/resources/mapfish-spring-config-output-formats.xml]: Fixed it

**Comment by sbrunner** [comment]
Sorry but I don't understand the goal of this pull request ...

**Comment by NielsCharlier** [comment]
Can this be merged?

File: core/src/main/java/org/mapfish/print/processor/map/CreateMapProcessor.java [hunk 2]
```diff
@@ -188,19 +192,46 @@ private URI createMergedGraphic(final File printDirectory,
...
+            com.lowagie.text.Document document = new com.lowagie.text.Document(
+                    new com.lowagie.text.Rectangle(width, height));
+            try {
+                PdfWriter writer = PdfWriter.getInstance(document, new FileOutputStream(mergedGraphic));
+                document.open();
+                PdfContentByte pdfCB = writer.getDirectContent();
+                Graphics g = pdfCB.createGraphics(width, height);                
+                drawGraphics(width, height, graphics, g);             
+                g.dispose();
+                document.close();
```
Comment (pvalsecc) [inline core/src/main/java/org/mapfish/print/processor/map/CreateMapProcessor.java]: Those two belong to a finally.

Comment (NielsCharlier) [inline core/src/main/java/org/mapfish/print/processor/map/CreateMapProcessor.java]: Made the change

**Pull request merged**